hnn_core/network.py

```diff
@@ -984,24 +1002,68 @@ def add_connection(self, src_gid, target_gid, loc, receptor,
             Synaptic delay in ms.
         lamtha : float
             Space constant.
+
+        N.B. Connections are stored in:
+        net.connectivity[idx]['gid_pairs'] : list of list
+            List of connected gids with the format:
+            [[src_gid, [target_gids, ...]], ...]
```

Comment:
one danger with list of list is that you might try to index `net.connectivity` by index of the list when you actually mean `src_gid`. Not sure if it's a real danger but if you did "dict of list" you wouldn't have this issue. The memory footprint might be slightly larger maybe?

Reply:
I'll checkout the memory change, can't imagine it'll be a huge increase. This might make checking redundant connections easier too.